tests/ssg_test_suite/oscap.py

```diff
@@ -550,6 +551,13 @@ def remediation(self):
         formatting['playbook'] = os.path.join(LogHelper.LOG_DIR,
                                               formatting['output_file'])
 
+        fix = xml_operations.find_fix_in_benchmark(self.datastream, self.benchmark_id, self.rule_id, fix_type='ansible')
+        if fix is None:
+            msg = (
+                'NO remediation was found for rule {}'.format(self.rule_id))
+            LogHelper.preload_log(logging.WARNING, msg, 'fail')
```

Comment:
How about making this an error?

Reply:
I like the idea. I considered doing in the first place but it requires additional changes to the code and I don't know how big it will be. I'll take a look and evaluate changing it.

Reply:
I decided to not throw error and treat the same as other cases by having only log message.